website/docs/r/event_trigger.html.markdown

```diff
@@ -0,0 +1,138 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: event_trigger"
+sidebar_current: "docs-mongodbatlas-resource-event-trigger"
+description: |-
+    Provides a Event Trigger resource.
+---
+
+# mongodbatlas_event_trigger
+
+`mongodbatlas_event_trigger` provides a Event Trigger resource. 
+
+## Example Usages
+
+### Example Usage: DATABASE with function
+```hcl
+resource "mongodbatlas_event_trigger" "test" {
+  project_id = "PROJECT ID"
+  app_id = "APPLICATION ID"
+  name = "NAME OF THE TRIGGER"
+  type = "DATABASE"
+  function_id = "FUNCTION ID"
+  disabled = false
+  config_operation_types = ["INSERT", "UPDATE"]
+  config_database = "DATABASE NAME"
+  config_collection = "COLLECTION NAME"
+  config_service_id = "SERVICE ID"
+  config_match = <<-EOF
+{
+  "updateDescription.updatedFields": {
+    "status": "blocked"
+  }
+}
+EOF
+  config_project = "{\"updateDescription.updatedFields\":{\"status\":\"blocked\"}}"
+  config_full_document = false
+  config_full_document_before = false
+  event_processors {
+    aws_eventbridge {
+      config_account_id = "AWS ACCOUNT ID"
+      config_region = "AWS REGIOn"
+    }
+  }
+}
+```
+
+### Example Usage: DATABASE with eventbridge
```

Comment:
Same, just make it a bit easier to read:
Example Usage: Database Trigger with EventBridge